### DISCUSSION TITLE
Und wer schreibt für das anstehende SAF TdSE Release eine GfSE Pressemitteilung?

### DISCUSSION THREAD

**Discussion by clalitsc** [GfSE/SAF-Specification#39] [Q&A]
Und wer schreibt für das anstehende SAF TdSE Release eine GfSE Pressemitteilung, die dann von Thomas Eilers auf der GfSE Startseite unter der Rubrik News veröffentlicht wird? Eine entsprechende Pressemitteilung könnte am SAF Plakat zu spontanen Flashmobs führen.
@GfSE/saf

**Comment by parkaneric** [comment]
Ich kann es _notfalls_ übernehmen. Mache sowas aber nicht allzu häufig ...
Hat irgendjemand einen Kontakt zu Kommunikationsprofis, die so etwas häufiger machen?

**Reply by clalitsc** [reply]
GfSE Fachbereich Öffentlichkeitsarbeit, Stefan Nerke <stefan.nerke@hood-group.com>
GfSE Fachbereich Mitgliederzufriedenheit, Alexander Efremoff <alexander.efremoff@gmail.com>
Produkt von Mitgliedern, für Mitglieder, über Mitglieder?

**Comment by parkaneric** [comment]
Post auf LinkedIn bzw. Xing wäre m.E. ebenso wichtig!

**Reply by clalitsc** [reply]
GfSE Fachbereich Öffentlichkeitsarbeit, Stefan Nerke <stefan.nerke@hood-group.com>, Stefan ist LinkedIn Content Admin

**Comment by parkaneric** [comment]
Stefan Nerke angeschrieben.
